packages/gdl-frontend/types.js

```diff
@@ -201,3 +203,20 @@ export type ChapterPointer = $ReadOnly<{
   chapterId: number,
   seqNo: number
 }>;
+
+type Messages = {
+  [messageId: string]: string | Function
+};
+
+type Catalog = {
+  messages: Messages
+};
+
+type Catalogs = {
+  [language: string]: Catalog
+};
+
+export type Locale = {
+  language?: string,
+  catalog?: Catalogs
```

Comment:
Disse to setter du som optionals, men strengt tatt må mann vel sette de? Så kanskje un-optional (nytt år :D) begge variablene?